packages/perseus/src/__testdata__/server-item-renderer.testdata.ts

```diff
@@ -359,3 +359,196 @@ export const itemWithLintingError: PerseusItem = {
     itemDataVersion: {major: 0, minor: 0},
     answer: null,
 };
+
+export const itemWithImages: PerseusItem = {
+    question: {
+        content: `_An example of the different images and their alignments._
+
+All articles in the main content of Articles and Exercises should be **centered**.
+
+A small portrait image:
+[[\u2603 image 1]]
+
+A large portrait Jpeg image:
+[[\u2603 image 2]]
+
+A small landscape image:
+[[\u2603 image 3]]
+
+A large portrait image:
+[[\u2603 image 4]]
+
+And what follows are _hints_...
+`,
```

Comment:
**code style nit:** doing this kind of template stuff in an object just looks weird. I would recommend, in order of preference:
- using `"` and `\n` like existing test data
- making a separate variable to store this content so it's not in the object.